internal/dao/types.go

```diff
@@ -120,7 +120,7 @@ type Controller interface {
 // Nuker represents a resource deleter.
 type Nuker interface {
 	// Delete removes a resource from the api server.
-	Delete(path string, cascade, force bool) error
+	Delete(path string, propagation string, force bool) error
```

Comment:
I think it would be best to used the k8s type here vs string ie Delete(string, *metav1.DeletionPropagation, bool). Makes the signature more to the point hmo and you can pass in nil vs "".

Reply:
I was on the fence between string and DeletionPropagation, primarily because using DeletionPropagation requires casting for the UI logic. You raise a good point about being able to pass in nil though, and I agree that the signature is clearer. Thanks for the input, I'll get this change made.